roles/common/tasks/ssl.yml

```diff
@@ -18,9 +18,14 @@
 - name: ssl directory
   file: dest=/opt/stack/ssl state=directory
 
-- name: openstack cert
-  template: src=openstack.crt dest=/opt/stack/ssl/openstack.crt mode=0644
-  tags: cert
+- name: install openstack SSL cert
+  template:
+    src: openstack.crt
+    dest: /opt/stack/ssl/openstack.crt
+    mode: 0644
+  tags:
+    - cert
+
```

Comment:
This is an unrelated change that was due to an earlier addition of this work touching this file. But since this is formatting the task how we want things formatted in the future, I'm leaving it in.